### ISSUE TITLE
Runtime error with Turkish culture

### ISSUE THREAD

**Issue by ErikEJ** [TrackableEntities/EntityFrameworkCore.Scaffolding.Handlebars#104]
see https://github.com/ErikEJ/EFCorePowerTools/issues/366

**Comment by ErikEJ** [comment]
Test (brittle, do not commit);

```
        [Theory]
        [InlineData(false, "en-US")]
        [InlineData(true, "en-US")]
        [InlineData(false, "tr-TR")]
        [InlineData(true, "tr-TR")]
        public void WriteCode_Should_Generate_Context_File(bool useDataAnnotations, string culture)
        {
            // Arrange
            Thread.CurrentThread.CurrentCulture = new System.Globalization.CultureInfo(culture);
            var options = ReverseEngineerOptions.DbContextOnly;
            var scaffolder = CreateScaffolder(options);

            // Act
            var model = scaffolder.ScaffoldModel(
                connectionString: Constants.Connections.SqlServerConnection,
                databaseOptions: new DatabaseModelFactoryOptions(),
                modelOptions: new ModelReverseEngineerOptions(),
                codeOptions: new ModelCodeGenerationOptions
                {
                    ContextNamespace = Constants.Parameters.RootNamespace,
                    ModelNamespace = Constants.Parameters.RootNamespace,
                    ContextName = Constants.Parameters.ContextName,
                    ContextDir = Constants.Parameters.ProjectPath,
                    UseDataAnnotations = useDataAnnotations,
                    Language = "C#",
                });

            // Assert
            var files = GetGeneratedFiles(model, options);
            object expectedContext = useDataAnnotations
                ? ExpectedContextsWithAnnotations.ContextClass
                : ExpectedContexts.ContextClass;

            var context = files[Constants.Files.CSharpFiles.DbContextFile];

            Assert.Equal(expectedContext, context);
        }

```

Fix:
```

        protected override IDictionary<string, string> GetPartialTemplates(
            LanguageOptions language = LanguageOptions.CSharp)
        {
            DbContextTemplateFiles.TryGetValue(Constants.DbContextImportTemplate, out TemplateFileInfo importFile);
            var importTemplateFile = FileService.RetrieveTemplateFileContents(
                importFile.RelativeDirectory, importFile.FileName);

            DbContextTemplateFiles.TryGetValue(Constants.DbContextCtorTemplate, out TemplateFileInfo ctorFile);
            var ctorTemplateFile = FileService.RetrieveTemplateFileContents(
                ctorFile.RelativeDirectory, ctorFile.FileName);

            DbContextTemplateFiles.TryGetValue(Constants.DbContextDbSetsTemplate, out TemplateFileInfo propertyFile);
            var propertyTemplateFile = FileService.RetrieveTemplateFileContents(
                propertyFile.RelativeDirectory, propertyFile.FileName);

            var templates = new Dictionary<string, string>
            {
                {
                    Constants.DbContextImportTemplate.ToLower(CultureInfo.InvariantCulture),
                    importTemplateFile
                },
                {
                    Constants.DbContextCtorTemplate.ToLower(CultureInfo.InvariantCulture),
                    ctorTemplateFile
                },
                {
                    Constants.DbContextDbSetsTemplate.ToLower(CultureInfo.InvariantCulture),
                    propertyTemplateFile
                },
            };
            return templates;
        }

```

**Comment by ErikEJ** [comment]
@tonysneed Any plans for a fix?

**Comment by tonysneed** [comment]
Fixed! :)

**Comment by ErikEJ** [comment]
Great, thanks! will you mention me when an updated Nuget package is published?

**Comment by tonysneed** [comment]
@ErikEJ Yes I will! Going to merge some PR’a and release a new version to NuGet this weekend.

**Comment by tonysneed** [comment]
@ErikEJ Published [v3.7.0](https://www.nuget.org/packages/EntityFrameworkCore.Scaffolding.Handlebars/3.7.0) of the NuGet package.
